urth-viz-query.html

```diff
@@ -437,7 +429,7 @@
         },
 
         _removeAgg: function(event) {
-            var i = getQueryIndex(event.currentTarget);
+            var i = event.currentTarget.getAttribute("data-query-index");
```

Comment:
I am getting an error here when I try to remove an aggregation. See above comment about adding attribute to `.agg-remove-btn`.